src/commands/commands.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @summary changes user access level (can only de-elevate)
+ * @param {import("../index").BotConfig} config bot config
+ * @param {import("../index").User} user message author
+ * @param {string} content incoming message content
+ * @returns {string}
+ */
+export const setAccessCommand = (config, user, content) => {
+    const [, userId, level] = /set (?:access|level)\s+(\d+|me)\s+(user|admin|dev)/.exec(content) || [];
+    if (!level) return "Please provide access level";
+
+    const uid = userId === "me" ? user.id : +userId;
+
+    if (config.debug) console.log({ userId, level, uid });
+
+    //TODO: move to user-based from id-based checks
+    const { adminIds, devIds } = config;
+
+    const changeMap = {
+        "user": () => {
+            adminIds.delete(uid);
+            devIds.delete(uid);
+        },
+        "admin": () => {
+            devIds.delete(uid);
+            adminIds.add(uid);
+        }
+    };
```

Comment:
not sure if you want to be able to set a mod access level here in case the tests require it in future

Reply:
an example would be the mod-command:
`what is the candidate score for 6` - fail successfully
`what is the candidate score for 584192` - should be 40
... etc.

Reply:
sorry, just realized admin here means mods+adminUids in the config. All good

Reply:
No worries :) No issue then?

Reply:
No issues 👍